### PR TITLE
Correct occurences of [&#91;...&#93;]

### LINKED ISSUE #1608: Link texts contain garbled text instead of target page title
Migrated content has some weird links such as 
`[&#91;\<topic://cpconbuildingaspnetwebservices>&#93;](https://msdn.microsoft.com/library/01dfc27c-c68e-4910-a0aa-5e4c2a766b0c)`

Links should be updated to have a page title as the link text instead of some internal keyword used on old internal systems and links should be updated to the new target on docs.microsoft.com (without locale).

e.g.
`[XML Web Services Using ASP.NET](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/ba0z6a33(v=vs.100))`

The list with the 53 links with this issue is attached to the issue.

[links.txt](https://github.com/dotnet/dotnet-api-docs/files/2756994/links.txt)

### PR DESCRIPTION
**Pull request by mikkelbu**
All occurences of `[&#91;...&#93;]` have been correct except:
* one occurence in xml\System.Runtime.Remoting.Contexts\IContributeObjectSink.xml
as it currently just links to https://docs.microsoft.com/en-us/previous-versions/
* ~~all occurences in xml\System.Xaml\XamlLanguage.xml
as these will be corrected in another PR~~

Edit: The occurences in xml\System.Xaml\XamlLanguage.xml seems fine.

Fixes #1608

### CODE REVIEW

File: xml/System.Runtime.Remoting.Contexts/SynchronizationAttribute.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
...
+>  There are two classes named `SynchronizationAttribute` : one in the <xref:System.Runtime.Remoting.Contexts> namespace, and the other in the <xref:System.EnterpriseServices> namespace. The <xref:System.EnterpriseServices.SynchronizationAttribute?displayProperty=nameWithType> class supports only synchronous calls, and can be used only with serviced components. (For more information on serviced components, see [Visual Studio 2005 Retired documentation](https://www.microsoft.com/download/details.aspx?id=55984).) The <xref:System.Runtime.Remoting.Contexts.SynchronizationAttribute?displayProperty=nameWithType> supports both synchronous and asynchronous calls, and can be used only with context bound objects. (For more information on context bound objects, see the <xref:System.ContextBoundObject> class.)
```
Comment (mikkelbu) [inline xml/System.Runtime.Remoting.Contexts/SynchronizationAttribute.xml]: Note that this just points to "Visual Studio 2005 Retired documentation"

File: xml/System.Net/HttpWebRequest.xml [hunk 2]
```diff
@@ -1445,7 +1445,7 @@ Both <xref:System.Net.HttpWebRequest> constructors are obsolete and should not b
 >  Your application cannot mix synchronous and asynchronous methods for a particular request. If you call the <xref:System.Net.HttpWebRequest.BeginGetRequestStream%2A> method, you must use the <xref:System.Net.HttpWebRequest.BeginGetResponse%2A> method to retrieve the response.  
   
 > [!NOTE]
->  This member outputs trace information when you enable network tracing in your application. For more information, see [&#91;\<topic://conUsingNetworkTracing>&#93;](https://msdn.microsoft.com/library/e993b7c3-087f-45d8-9c02-9dded936d804).  
+>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](https://docs.microsoft.com/dotnet/framework/network-programming/network-tracing).
```
Comment (mairaw) [inline xml/System.Net/HttpWebRequest.xml]: Let's use the ~ links to get build time validation and have the links work on offline books as well

```suggestion
>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](~/docs/framework/network-programming/network-tracing.md).  
```

File: xml/System.Net/WebRequest.xml [hunk 3]
```diff
@@ -742,7 +742,7 @@
  The <xref:System.Net.WebRequest.Create%2A> method uses the `requestUriString` parameter to create a <xref:System.Uri> instance that it passes to the new <xref:System.Net.WebRequest>.  
   
 > [!NOTE]
->  This member outputs trace information when you enable network tracing in your application. For more information, see [&#91;\<topic://conUsingNetworkTracing>&#93;](https://msdn.microsoft.com/library/e993b7c3-087f-45d8-9c02-9dded936d804).  
+>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](https://docs.microsoft.com/dotnet/framework/network-programming/network-tracing).
```
Comment (mairaw) [inline xml/System.Net/WebRequest.xml]:
```suggestion
>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](~/docs/framework/network-programming/network-tracing.md).
```

File: xml/System.Net/WebRequest.xml [hunk 4]
```diff
@@ -834,7 +834,7 @@
  .NET includes support for the `http://`, `https://`, `ftp://`, and `file://` URI schemes. Custom <xref:System.Net.WebRequest> descendants to handle other requests are registered with the <xref:System.Net.WebRequest.RegisterPrefix%2A> method.  
   
 > [!NOTE]
->  This member outputs trace information when you enable network tracing in your application. For more information, see [&#91;\<topic://conUsingNetworkTracing>&#93;](https://msdn.microsoft.com/library/e993b7c3-087f-45d8-9c02-9dded936d804).  
+>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](https://docs.microsoft.com/dotnet/framework/network-programming/network-tracing).
```
Comment (mairaw) [inline xml/System.Net/WebRequest.xml]:
```suggestion
>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](~/docs/framework/network-programming/network-tracing.md).  
```

File: xml/System.Runtime.Remoting.Contexts/SynchronizationAttribute.xml (hunk 1 above)
Comment (mairaw) [inline xml/System.Runtime.Remoting.Contexts/SynchronizationAttribute.xml]: Let's just remove that from the sentence then

```suggestion
>  There are two classes named `SynchronizationAttribute` : one in the <xref:System.Runtime.Remoting.Contexts> namespace, and the other in the <xref:System.EnterpriseServices> namespace. The <xref:System.EnterpriseServices.SynchronizationAttribute?displayProperty=nameWithType> class supports only synchronous calls, and can be used only with serviced components. The <xref:System.Runtime.Remoting.Contexts.SynchronizationAttribute?displayProperty=nameWithType> supports both synchronous and asynchronous calls, and can be used only with context bound objects. (For more information on context bound objects, see the <xref:System.ContextBoundObject> class.)  
```

**Comment by mikkelbu** [comment]
@mairaw I've applied all the suggestions.

**Comment by mairaw** [comment]
I realized I also missed one of your earlier comments. I did find some link targets in previous-versions for that link in IContributeObjectSink but it might be better to just delete that sentence. Would you like to take care of that link here too? 

image

**Comment by mikkelbu** [comment]
Ok. I've removed the sentence.

**Pull request merged**